presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -193,6 +193,11 @@ Map Aggregate Functions
    Returns the union of all the input maps. If a key is found in multiple
    input maps, that key's value in the resulting map comes from an arbitrary input map.
 
+.. function:: map_union_sum(x(K,V)) -> map(K,V)
+
+      Returns the union of all the input maps SUM'ing the values of matching keys in all
```

Comment:
"SUM'ing" should be "summing"? 
And I think in this case, since this is not a standard SQL function, we should add an example illustrating what this function does.

Reply:
Done